DocsDevREADME.md

```diff
@@ -212,7 +212,7 @@ end tell
    - If you crop the bottom or top, use the `bottom-cropped` or `top-cropped` class on the image. In some cases the
      class may not be necessary if there is adequate spacing below. When text continues below or right above you will need
      the class.
-- If you crop the image, don't use the `box-shadow` role. And vice versa.
```

Comment:
I think `box-shadow` is incorrect here. The first section says
> Use `box-shadow` only when an image isn't captured in the manner documented below.

This section is "the manner documented below," and other uses of screenshots use `shadowed`.